src/enums/action-types.ts

```diff
@@ -1,5 +1,16 @@
-export enum ActionTypes {
-  ADDED = 'added',
-  EDITED = 'edited',
-  REMOVED = 'removed'
+/**
+ * Enums for action types, used by:
+ * - Action Chip (shared component)
+ * - Name Translations (Edit UI)
+ * - People And Roles (Edit UI)
+ * - Share Structure (shared component and Edit UI)
+ */
+ export enum ActionTypes {
+  ADDED = 'ADDED',
+  ADDRESS_CHANGED = 'ADDRESS CHANGED',
+  CORRECTED = 'CORRECTED',
+  EDITED = 'EDITED',
+  EMAIL_CHANGED = 'EMAIL CHANGED',
+  NAME_CHANGED = 'NAME CHANGED',
+  REMOVED = 'REMOVED'
 }
```

Comment:
This is the ActionTypes enum as used by Edit UI (which is actually the only UI that uses this, as well as some shared components).